src/silo/query_engine/query_engine.cpp

```diff
@@ -26,6 +26,11 @@
 #include "silo/database.h"
 #include "silo/query_engine/query_result.h"
 
+#define CHECK_SILO_QUERY(condition, message)    \
```

Comment:
Do we want to extend the macro, that it can also work with other exceptions (namely extending it with one more argument), or do we want to keep it specific to silo queries?